contentcuration/contentcuration/frontend/shared/views/StudioImmersiveModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+
+  <div
+    v-if="value"
+    class="modal-wrapper"
+    data-testid="modal-wrapper"
+    :style="{ backgroundColor: $themeTokens.surface }"
+  >
+    <KToolbar
+      textColor="white"
+      :style="{ backgroundColor: $themePalette.grey.v_900 }"
+    >
+      <template #icon>
+        <KIconButton
+          icon="close"
+          :ariaLabel="$tr('close')"
+          :tooltip="$tr('close')"
+          color="white"
+          data-test="close"
+          @click="$emit('input', false)"
+        />
+      </template>
+
+      <template #default>
+        <span class="notranslate toolbar-title">
+          <slot name="header">{{ title }}</slot>
+        </span>
+      </template>
+
+      <template #actions>
+        <slot name="action"></slot>
+      </template>
+    </KToolbar>
+
+    <StudioOfflineAlert :offset="64" />
```

Comment:
This is too large offset causing extra space between the toolbar and offline bar.

Also, when offline, _Download channel summary_ button shouldn't be below the offline bar, not even partially.